pimd/pim6_cmd.c

```diff
@@ -1572,6 +1572,125 @@ DEFPY (show_ipv6_pim_nexthop_lookup,
 	return CMD_SUCCESS;
 }
 
+DEFPY (show_ipv6_multicast,
+       show_ipv6_multicast_cmd,
+       "show ipv6 multicast [vrf NAME]",
+       SHOW_STR
+       IPV6_STR
+       VRF_CMD_HELP_STR
+       "Multicast global information\n")
+{
+	struct vrf *v;
+	struct pim_instance *pim;
+
+	v = vrf_lookup_by_name(vrf ? vrf : VRF_DEFAULT_NAME);
```

Comment:
I believe it will never be NULL since we fall back to default vrf, right?

Reply:
Fixed